final/introduction.tex

```diff
@@ -1,6 +1,33 @@
 \chapter{Introduction}
 \label{cha:introduction}
 
+Besides general purpose languages such as Java and C, nowadays there are also a
+lot of domain specific languages available. Domain specific languages can
+provide a lot of expressive power in a particular problem domain, thereby
+significantly easing the burden on the programmer. Well known examples of such
+DSLs include HTML for creating websites and SQL for dealing with relational
+data.
+
+At TU Delft significant effort has been spent to research and develop better
+domain specific languages. To facilitate prototyping and creation of DSLs the
+Spoofax Language Workbench was developed and first published in
+2010\cite{Kats10a}. Since then Spoofax has had several releases and will soon
+be arriving at version number 2.0.
+
+The description below is taken from \href{http://spoofax.org}{spoofax.org}:
+``The Spoofax Language Workbench supports the definition of all aspects of
+textual languages using high-level, declarative meta-languages. From a
+language definition using these meta-languages, Spoofax generates full-featured
+Eclipse and IntelliJ editor plugins, as well as a command-line interface.''
+
+To facilitate prototyping DSLs even better by adding the ability of exploratory
```

Comment:
You can also read the preface, problem description, infosheet and/or project plan for inspiration to improve and add to this paragraph.